DROD/CharacterDialogWidget.cpp

```diff
@@ -189,6 +190,9 @@ const UINT LIST_LINE_HEIGHT = 22;
 const SURFACECOLOR PaleRed = {255, 192, 192};
 
 std::map<UINT, UINT> CCharacterDialogWidget::speechLengthCache;
+const UINT CCharacterDialogWidget::INDENT_PREFIX_SIZE = 6;
+const UINT CCharacterDialogWidget::INDENT_TAB_SIZE = 3;
+const UINT CCharacterDialogWidget::INDENT_IF_CONDITION_SIZE = 5;
```

Comment:
If these are purely hardcoded constants, consider using a [`constexpr` variable](https://en.cppreference.com/w/cpp/language/constexpr) (C++11), or an `inline constexpr` variable if C++17 support is available. Just makes it more obvious and has a stronger guarantee that the value is purely known at compile-time.